samples/dashboards/demo/datagrid-mathmodifier/demo.html

```diff
@@ -5,17 +5,16 @@
 <script src="https://code.highcharts.com/modules/draggable-points.js"></script>
 <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
 
-<div id="container">
-
-<pre id="csv">Day,EUR,Rate
-15,11,1.0876
-16,23,1.0881
-17,15,1.0829
-18,27,1.0813
-19,13,1.0808</pre>
-
-<p>
-    The dashboard demonstrates some capabilities of the MathModifier. The last
-    column in the table is calculated automatically by a defined columnFormula
-    option with the formula term "B1*C1".
+<div id="container"></div>
+<p class="highcharts-description">
+    The dashboard demonstrates some capabilities of the MathModifier. The
+    last column in the table is calculated automatically by a defined
+    columnFormula option with the formula term "B1*C1".
```

Comment:
I would add that you are not able to modify the formula in the table only other cells. Which is also reflected on the chart.